packages/core/src/browser/tree/tree-widget.tsx

```diff
@@ -569,7 +589,7 @@ export class TreeWidget extends VirtualWidget implements StatefulWidget {
         this.model.openNode();
     }
 
-    protected handleClickEvent(node: TreeNode | undefined, event: MouseEvent): void {
+    protected handleClickEvent(node: TreeNode | undefined, event: React.MouseEvent<HTMLElement>): void {
```

Comment:
Would it be possible to have `handleLeft/Right/Up/Down/Enter` with the React events instead of the native ones? Just to have symmetric API. See: `handleClickEvent/handleDblClickEvent/handleContextMenuEvent`?

Reply:
For now I've created a follow up issue for that. https://github.com/theia-ide/theia/issues/2254